src/Conversions.cc

```diff
@@ -1792,3 +1792,35 @@ msgs::Plugin ignition::gazebo::convert(const sdf::Plugin &_in)
 
   return result;
 }
+
+//////////////////////////////////////////////////
+template<>
+IGNITION_GAZEBO_VISIBLE
+msgs::Plugin_V ignition::gazebo::convert(const sdf::Plugins &_in)
+{
+  msgs::Plugin_V result;
+  for (const sdf::Plugin &plugin : _in)
+  {
+    result.add_plugins()->CopyFrom(convert<msgs::Plugin>(plugin));
+  }
+  return result;
+}
+
+//////////////////////////////////////////////////
+template<>
+IGNITION_GAZEBO_VISIBLE
+sdf::Plugin ignition::gazebo::convert(const msgs::Plugin &_in)
```

Comment:
We should add tests for these new conversions

Reply:
Done in [afd30fb](https://github.com/gazebosim/gz-sim/pull/1352/commits/afd30fbb79c7c81cf9d7bb056179fa6cd92a0d94)